input_data.go

```diff
@@ -6,7 +6,9 @@ type Modifier int
 
 var (
```

Comment:
The exported `Mouse` variable feels strange, as its type (`mouse`) is not exported, but the `mouse` type contains exported members (e.g. `mouse.X`, `mouse.Action`, etc). Is `engo.Mouse` intended to be exported? If so, its type should probably be exported as well, so it may be included in documentation.